source/runtime_gui.cpp

```diff
@@ -3412,7 +3412,8 @@ void reshade::runtime::draw_variable_editor()
 			{
 			case reshadefx::type::t_bool:
 				get_uniform_value(variable, reinterpret_cast<bool *>(value.as_uint), variable.type.components());
```

Comment:
Best to not cast to boolean here (and line 3416 below), but rather simply get the integer value as-is and compare that against zero.